FTLinearActivityIndicator.podspec

```diff
@@ -22,7 +22,7 @@ Pod::Spec.new do |s|
   s.ios.deployment_target = '8.0'
   s.swift_version = '5.0'
 
-  s.source_files = 'FTLinearActivityIndicator/Classes/**/*'
+  s.source_files = 'Sources/FTLinearActivityIndicator/Classes/**/*'
```

Comment:
Why move the file if you can just specify a different directory in the SwiftPM config?

Reply:
@Zandor300 would you be able to adjust the source_files fairly quickly, or can I be of some help?

Reply:
@futuretap Thank you for the quick response.